src/07-projects/projects.apib

```diff
@@ -111,6 +111,22 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + billing_method (object, required)
+            + One Of
+                + Properties
+                    + method: `fixed` (string, required)
+                    + amount (Money, required)
+                + Properties
+                    + method: `time_material` (string, required)
+                    + hourly_rate (object, required)
```

Comment:
Q: Could we call this something that doesn't include the period? I'd rather not have `hourly_rate`, `weekly_rate`, etc, in the future, if we decide to implement that. Maybe just `rate`?